server/src/main/java/org/spine3/server/entity/storage/reflect/Column.java

```diff
@@ -0,0 +1,207 @@
+        return resullt;
+    }
+
+    /**
+     * Retrieves the name of the property which represents the Storage Field.
+     *
+     * <p>For example, if the getter method has name "isArchivedOrDeleted", the returned value is
+     * "archivedOrDeleted".
+     *
+     * @return the name of the property reflected by this object
+     */
+    public String getName() {
+        return name;
+    }
+
+    /**
+     * Shows if the Storage Field may return {@code null}s.
+     *
+     * @return {@code true} if the getter method was annotated as {@link Nullable},
+     * {@code false} otherwise
+     */
+    public boolean isNullable() {
+        return nullable;
+    }
+
+    /**
+     * Retrieves the Storage Field value from the given {@link Entity}.
+     *
+     * @param source the {@link Entity} to get the Fields from
+     * @return the value of the Storage Field represented y this instance of {@code Column}
+     */
+    public T getFor(Entity<?, ?> source) {
+        try {
+            @SuppressWarnings("unchecked")
+            final T result = (T) getter.invoke(source);
+            if (!nullable) {
+                checkNotNull(result, format("Not null getter %s returned null.", getter.getName()));
+            }
+            return result;
+        } catch (IllegalAccessException | InvocationTargetException e) {
+            throw new IllegalStateException(
+                    format("Could not invoke getter of property %s from object %s",
+                           getName(),
+                           source),
+                    e);
+        }
+    }
+
+    /**
+     * Retrieves the Storage Field value from the given {@link Entity}.
+     *
+     * <p>The value is wrapped into a special container, which bears information about the Field's
+     * metadata
+     *
+     * @param source the {@link Entity} to get the Fields from
+     * @return the value of the Storage Field represented y this instance of {@code Column} wrapped
+     * into {@link MemoizedValue}
+     * @see MemoizedValue
+     */
+    public MemoizedValue<T> memoizeFor(Entity<?, ?> source) {
+        final T value = getFor(source);
+        final MemoizedValue<T> result = new MemoizedValue<>(this, value);
+        return result;
+    }
+
+    /**
+     * @return the type of the Storage Field
+     */
+    @SuppressWarnings("unchecked")
+    public Class<T> getType() {
+        return (Class<T>) getter.getReturnType();
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+
+        Column<?> column = (Column<?>) o;
+
+        return getter.equals(column.getter);
+    }
+
+    @Override
+    public int hashCode() {
+        return getter.hashCode();
+    }
+
+    /**
+     * A memoized in a point in time value of the Storage Field.
```

Comment:
`A value of a Storage Field saved at some point in time.`

BTW, I'd add something else, since that's too short — basically you just repeated the class name (i.e. `Column.MemoizedValue`.

Also, since you are using it from other classes, I'd probably move it to an upper level.